design/api.go

```diff
@@ -17,13 +17,13 @@ var _ = API("fuseml", func() {
 		Services("runnable", "openapi")
 
 		// List the Hosts and their transport URLs.
-		Host("localhost", func() {
```

Comment:
I would rather keep this localhost, which is a safer default value (at least for me, when running locally) and override it in the container image entrypoint as 0.0.0.0. It also creates fewer changes overall :)

Reply:
I have updated the code with the addition of the `prod` host that will listen on `0.0.0.0` and kept the default `localhost` as the `dev` host.

Reply:
thanks